src/test/java/org/vamae/controller/GameBoardControllerTest.java

```diff
@@ -0,0 +1,45 @@
+package org.vamae.controller;
+
+import static org.junit.jupiter.api.Assertions.*;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.vamae.entity.GameBoard;
+import org.vamae.enums.Piece;
+import org.vamae.exceptions.PieceOutOfBoardException;
+
+class GameBoardControllerTest {
+        private GameBoardController gameBoardController;
+        private GameBoard gameBoard;
+        @BeforeEach
+        public void setUp() {
+            gameBoard = new GameBoard(7, 6);
+            gameBoardController = new GameBoardController(gameBoard);
+        }
+
+        @Test
+        public void isInBoardValidCoordinates() {
```

Comment:
Tests should be named "test<method name><case>". Add "test" before each method name